sklearn/decomposition/tests/test_nmf.py

```diff
@@ -426,7 +561,7 @@ def test_nmf_multiplicative_update_sparse():
     for beta_loss in (-1.2, 0, 0.2, 1.0, 2.0, 2.5):
         # Reference with dense array X
         W, H = W0.copy(), H0.copy()
-        W1, H1, _ = non_negative_factorization(
+        W1, H1, *_ = non_negative_factorization(
```

Comment:
Naive question: why is `*_` introduced here and in similar calls to `non_negative_factorization`?

Reply:
leftover of changes in non_negative_factorization that were not kept. I removed them.